src/topics/group-generator/test-group-generator.ts

```diff
@@ -127,14 +127,17 @@ export const testGeneratorGenerations = {
   testGeneration1_0: {
     name: "test-generator1",
     timestamp: 1,
+    lastGenerationDuration: 1,
```

Comment:
Are you sure it's well tested? What are these values?

Reply:
Here I put random value because it's only used in the `dynamodb-group-generator-store.test` for db storing tests.
However I agree with you it miss some test regarding the API, thanks!